File: Prime.java
import java.util.*;
import java.lang.*;
public class Prime
{
	public static void main(String args[])
	{
		int a,count=0,i,b;
		Scanner sc=new Scanner(System.in);
		a=sc.nextInt();
		b=a%10;
		for(i=1;i<=a;i++)
		{
		    if(a%i==0)
		    {
		        count++;
		    }
		}
		if(count==2)
		{
		    System.out.println("yes");
		}
		else
		{
		    System.out.println("no");
		}
	}
}

File: prime.cpp
#include <iostream>
using namespace std;
int main()
{
	int n,c=0,i;
	cin>>n;
	for(i=1;i<=n;i++)
	{
		if(n%i==0)
		{
			c=c+1;
		}
	}
	if(c==2)
		cout<<"yes";
	else
		cout<<"no";
	return 0;
}
